dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/chart/conditional-formatting.md

```diff
@@ -0,0 +1,96 @@
+
+The following [series types](series.md) support conditional formatting:
+
+* Bar
+* Point/Line
+* Area
+* Bubble 
+* Range Bar
+
+## Supported Format Rules
+
+You can use the following data in rule calculations:
+
+- [measures](../../binding-dashboard-items-to-data/binding-dashboard-items-to-data-in-the-web-dashboard.md) from the **Values** section 
+- [dimensions](../../binding-dashboard-items-to-data/binding-dashboard-items-to-data-in-the-web-dashboard.md) from the **Arguments**/**Series** section 
+- [hidden measures](../../binding-dashboard-items-to-data/hidden-data-items.md) 
+
+Format conditions that can be applied to different data item types are as follows:
+* numeric 
+	* **Value**
+	* **Top-Bottom**
+	* **Average**
+	* **Expression** 
+	* **Color Ranges**
+	* **Gradient Ranges**
+* string 
+	* **Value** (with the condition type set to _Equal To_, _Not Equal To_ or _Text that Contains_)
+	* **Expression**
+* date-time 
+	* **Value**
+	* A **Date Occurring** (for dimensions with a continuous date-time group interval)
+	* **Expression**
+	* **Color Ranges**
+	* **Gradient Ranges**
+
+Refer to the following topic for more information about format condition types: [Conditional Formatting in Web Dashboard](../../appearance-customization/conditional-formatting.md).
+
+## Create and Edit a Format Rule   
+
+You can create and edit format rules in the **Conditional Formatting** section that is located in the following places:
+
+* The dashboard item's [Options](../../ui-elements/dashboard-item-menu.md) menu
+
+*  [data item menu](../../ui-elements/data-item-menu.md)
+
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting in Web Dashboard](../../appearance-customization/conditional-formatting.md).
+
+## Chart-Specific Format Condition Settings
+
+Specify appearance settings and set the condition's value to create a format rule. Available settings depend on the selected format condition type.
+
+The image below displays the **Value** rule settings. The condition colors points/bars if their values exceed 1400.
+
+![web-chart-greater-than-dialog-example](../../../../images/web-chart-greater-than-dialog-example.png)
+
+You can apply one of the predefined colors or set a custom color for this condition.
+
+Use the **Apply to chart elements** button group to apply a rule on points or lines.
+
+* **Point**: A rule applies to the data-points such as bars, points and bubbles.
+* **Line / Area**: A rule applies to lines and areas.
+
+If you select **Line / Area**, the format rule applies to the line when at least one line point meets the rule's condition: 
+
+![web-line-chart-with-applied-value-format-rule](../../../../images/web-line-chart-with-applied-value-format-rule.png)
+
+Go to the rule's **Legend** section and set the **Caption** field to specify the legend's text. It enables the **Display in Legend** option and the Chart item displays information about the applied rule in the legend.
```

Comment:
`It enables the **Display in Legend** option and the Chart item displays information about the applied rule in the legend.`
Does the "Display in Legend" option display information about the applied rule? If so, can we say:

`It enables the **Display in Legend** option, so that the Chart item displays information about the applied rule in the legend.`